backend/migrations/versions/59bb2488a76b_insert_roles.py

```diff
@@ -0,0 +1,41 @@
+"""insert roles
+
+Revision ID: 59bb2488a76b
+Revises: 4ba3479cb8df
+Create Date: 2024-10-16 16:55:42.324525
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "59bb2488a76b"
+down_revision: Union[str, None] = "4ba3479cb8df"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.bulk_insert(
+        sa.table(
+            "roles",
+            sa.Column("id", sa.Integer(), nullable=False),
+            sa.Column("name", sa.String(length=80), nullable=False),
+        ),
+        [
+            {"id": 1, "name": "participant"},
+            {"id": 2, "name": "volunteer"},
+            {"id": 3, "name": "admin"},
+        ],
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
```

Comment:
Should we have a command to remove the roles we added in downgrade?

Reply:
yep let's do that.